camel/agents/chat_agent.py

```diff
@@ -12,8 +12,9 @@
 # limitations under the License.
 # =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
 from dataclasses import dataclass
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
+import openai
```

Comment:
Chat agent should not depend on openai. This is a violation of the dependency inversion principle.